src/interface/blockchain.cpp

```diff
@@ -99,7 +99,9 @@ void blockchain::fetch_transaction(server_node& node, const message& request,
 
     // The response is restricted to confirmed transactions.
     // This response excludes witness data so as not to break old parsers.
-    node.chain().fetch_transaction(hash, true, false,
+    const auto require_confirmed = false; // FIXME: true
```

Comment:
Database appears to never think txs are confirmed at the moment, so this FIXME indicates a temporary workaround that will be reverted.

Reply:
These hacks are now gone, after merging latest database/blockchain PRs.